games/a.yaml

```
@@ -663,6 +663,22 @@
   added: '2025-11-06'
   updated: '2025-11-06'
 
+- name: Alpha Waves Classic
```

```suggestion
- name: Alpha-Waves / Continuum
```
I don't see "classic" mentioned